lib/index.js

```diff
@@ -243,44 +245,26 @@ Segment.prototype.ampId = function(ctx) {
  * @param {Function} fn
  */
 
-Segment.prototype.send = function(path, msg, fn) {
+Segment.prototype.enqueue = function(path, msg, fn) {
   var url = 'https://' + this.options.apiHost + path;
-  fn = fn || noop;
-  var self = this;
-
-  // msg
+  var headers = { 'Content-Type': 'text/plain' };
   msg = this.normalize(msg);
+  this.debug('enqueueing');
 
-  // send
+  var self = this;
   if (this.options.retryQueue) {
-    var headers = { 'Content-Type': 'text/plain' };
     this._lsqueue.addItem({
       url: url,
       headers: headers,
       msg: msg
     });
-  } else if (this.options.beacon && navigator.sendBeacon) {
-    // Beacon returns false if the browser couldn't queue the data for transfer
-    // (e.g: the data was too big)
-    if (navigator.sendBeacon(url, json.stringify(msg))) {
-      self.debug('beacon sent %o', msg);
-      fn();
-    } else {
-      self.debug('beacon failed, falling back to ajax %o', msg);
-      sendAjax();
-    }
   } else {
-    sendAjax();
-  }
-
-  function sendAjax() {
-    // Beacons are sent as a text/plain POST
-    var headers = { 'Content-Type': 'text/plain' };
     send(url, msg, headers, function(err, res) {
-      self.debug('ajax sent %o, received %o', msg, arguments);
-      if (err) return fn(err);
```

Comment:
this fn is only ever called in tests (which is true of how it worked before as well) — should i refactor?